src/maven/forms/msmsscoringsettings.ui

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Don't worry too much about this file - it was automatically generated using the `Qt Creator` IDE. In general, `.ui` files are used by `Qt` projects to store GUI details, and to generate necessary header and source files that plug into the Qt system.